hack/update-import-aliases.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
minor nit; i know lots of the script was probably copy/pasted, but should you update the year on a number of these files to be 2019? Whats the protocol on when to update it if you are, for instance, copy/pasting/editing an existing script?

Reply:
Fixed